Sources/EdgeNetworkHandlers/EdgeHit.swift

```diff
@@ -0,0 +1,29 @@
+//
+// Copyright 2020 Adobe. All rights reserved.
+// This file is licensed to you under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may obtain a copy
+// of the License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under
+// the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+// OF ANY KIND, either express or implied. See the License for the specific language
+// governing permissions and limitations under the License.
+//
+
+import AEPCore
+import Foundation
+
+/// Struct which represents an Edge hit
+struct EdgeHit {
+    /// The Edge configuration identifier
+    let configId: String
+
+    /// Unique identifier for this hit
+    let requestId: String
+
+    /// The `EdgeRequest` for the corresponding hit
+    let request: EdgeRequest
+
+    /// The `Event` associated with the `EdgeHit`
+    let event: Event
```

Comment:
Is this event used for anything else other than logging on fail here https://github.com/adobe/aepsdk-edge-ios/pull/103/files#diff-1f8b1d6008249e9d5807a9cdd4264cbcb5fdaee71d22df628697818ddaeeb39cR111-R117? 
I am thinking the Edge hit will contain a list of events in the future, but I wonder if this is required in the EdgeHit at all, we should be able to tie the responses back based on the requestId and the queued waiting events. If not required, let's remove it from here.